Functions/xASL_bids_Dicom2JSON.m

```diff
@@ -115,7 +115,8 @@
 	% this should not give an error, but multiple ASL scale slopes should give an error
 	% Check if FLAIR occurs
 	isASL = true; % by default, assume these are ASL DICOMs
-	if ~isempty(findstr(FileList{1},'FLAIR')) || ~isempty(findstr(FileList{1},'T1'))
```

Comment:
Same as above, replace by ~isempty(regexpi())

Reply:
Same as above - was completely deleted.

Reply:
> > * did you check that the names are equal to those used in BIDS?
> 
> Yes, T2 is similar to T1. So fine in BIDS and legacy. T1c is difficult, in BIDS, it is sub..._ce-_t1.nii . This will require more complex changes to BIDS import and BIDS2Legacy. I have created a new issue for that, but for the DICOM2Legacy and Legacy processing, I have inserted it already in the simplified T1c form.
> 
> > * we need to make sure that these new scantypes are only added when needed (to avoid having verbose T2-related logging every time ExploreASL runs). Likewise, for the QC folders?
> 
> Sure - will be added in the import only when specified.
> 
> > * Also see my in-line comments, I would rewrite to ensure that all the new T2/T1c code is together, since it will be used very rarely.
> 
> Inline comments answered and implemented accordingly - see the last commit.

`T2` should be `T2w`. I know we don't have this for `T1`, but would be good to phase to BIDS as good as possible.

Reply:
I made a new issue for that. Good to be close to BIDS, but bad to have T1.nii and T2w.nii. We can synchronize both in the future. Though when thinking about it. We might not need to as the legacy format will be phased out once we move to BIDS, so I would keep this as it is T1.nii and T2.nii.